libr/parse/parse.c

```diff
@@ -325,9 +327,15 @@ static int filter(RParse *p, RFlag *f, char *data, char *str, int len, bool big_
 									break;
 								}
 								memmove (ptr_left, ptr_esc, copied_len);
-								sprintf (ptr_left + copied_len, "%s%s",
-										ansi_found && ptr_right - ptr_end + 1 >= 4 ? "\x1b[0m" : "",
-										ptr_right + 1);
+								dptr_left = strcpy (ptr_left + copied_len,
```

Comment:
define and assign the variables here, no need to define them on top of the while